Gaijinizer patch/Map097.txt

```diff
@@ -961,9 +1010,9 @@ have a drizzle if it is gentle once.#MTLed
 > CONTEXT: Map139/events/2/pages/0/460/Dialogue/kazuya
 > CONTEXT: Map139/events/3/pages/0/461/Dialogue/kazuya
 > CONTEXT: Map139/events/4/pages/0/461/Dialogue/kazuya
-Besides, if she was attacked by M.O.W.'s
-"biological weapons". That's just a humble
-.........#MTLed
+Besides, if she's attacked by that M.O.W.
+biological weapon, that personality of hers will
+mean she won't stand a chance.#RonIffy
```

Comment:
Flow:
>_**Besides, if she's attacked by M.O.W's
biological weapon, that personality of hers
means she won't stand a chance.**_